Client.Linq/Internal/Expressions/String/EndsWith.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions.String
+{
+    internal class EndsWith : IExpressionPart
+    {
+        private readonly IEnumerable<IExpressionPart> expressionParts;
+        private readonly IEnumerable<IExpressionPart> expressionParts2;
+
+        internal EndsWith(IEnumerable<IExpressionPart> expressionParts, IEnumerable<IExpressionPart> expressionParts2)
+        {
+            this.expressionParts = expressionParts;
+            this.expressionParts2 = expressionParts2;
+        }
+
+        public void AppendFlux(StringBuilder builder)
+        {
+            builder.Append("strings.hasSuffix(v: ");
+            foreach (var expressionPart in expressionParts) expressionPart.AppendFlux(builder);
+            builder.Append(", suffix: ");
+            foreach (var expressionPart in expressionParts2) expressionPart.AppendFlux(builder);
+            builder.Append(")");
+        }
+    }
```

Comment:
Please use same naming as is in Flux documentation:

```suggestion
        private readonly IEnumerable<IExpressionPart> _value;
        private readonly IEnumerable<IExpressionPart> _suffix;

        internal EndsWith(IEnumerable<IExpressionPart> value, IEnumerable<IExpressionPart> suffix)
        {
            _value = value;
            _suffix = suffix;
        }

        public void AppendFlux(StringBuilder builder)
        {
            builder.Append("strings.hasSuffix(v: ");
            foreach (var expressionPart in _value) expressionPart.AppendFlux(builder);
            builder.Append(", suffix: ");
            foreach (var expressionPart in _suffix) expressionPart.AppendFlux(builder);
            builder.Append(")");
        }
```